test/evergreen/setup_spawn_host.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# The script is used to setup the environment for a spawn host. Currently the script 
+# unzips the test artefacts, prepend the toolchain path to the PATH environmen and exports the
+# LD_LIBRARY_PATH.
+# 
+# Once setup, the script will notify the user who spawned the host that it is finished.
+#
+cd $HOME
```

Comment:
You may want to set error on failures or add `|| exit 1` This is applicable for the other `cd` commands.

Reply:
done!